backend/danswer/connectors/linear/connector.py

```diff
@@ -0,0 +1,209 @@
+import os
+from datetime import datetime
+from datetime import timezone
+from typing import Any
+
+import requests
+
+from danswer.configs.app_configs import INDEX_BATCH_SIZE
+from danswer.configs.constants import DocumentSource
+from danswer.connectors.interfaces import GenerateDocumentsOutput
+from danswer.connectors.interfaces import LoadConnector
+from danswer.connectors.interfaces import PollConnector
+from danswer.connectors.interfaces import SecondsSinceUnixEpoch
+from danswer.connectors.models import Document
+from danswer.connectors.models import Section
+from danswer.utils.logger import setup_logger
+
+logger = setup_logger()
+
+_NUM_RETRIES = 5
```

Comment:
shared connector per doc retries config?

Reply:
Hmm, I'm personally more in-favor of allowing each connector to set this, since it depends on the reliability of the API